Tracking/include/Tracking/Reco/TrackExtrapolatorTool.h

```diff
@@ -51,16 +51,32 @@ class TrackExtrapolatorTool {
    @return optional with BoundTrackParameters
   */
 
+  using PropagatorOptions =
+      typename propagator_t::template Options<ActionList, AbortList>;
+
   std::optional<Acts::BoundTrackParameters> extrapolate(
       const Acts::BoundTrackParameters pars,
       const std::shared_ptr<Acts::Surface>& target_surface) {
     // Just to make it explicit
     bool boundaryCheck = false;
```

Comment:
this will make clang complain, as it's unused in the current state
```suggestion
    // bool boundaryCheck = false;
```